.github/scripts/trymerge.py

```diff
@@ -1080,15 +1080,15 @@ def merge_into(
         skip_mandatory_checks: bool = False,
         dry_run: bool = False,
         comment_id: Optional[int] = None,
-        ignore_current_checks: Optional[List[str]] = None,
```

Comment:
As the flag is called `ignore_current`, let also use the same name here instead of `skip_current_checks` for consistency